include/lsst/afw/image/PhotoCalib.h

```diff
@@ -502,6 +502,21 @@ class PhotoCalib : public table::io::PersistableFacade<PhotoCalib>, public table
  */
 std::shared_ptr<PhotoCalib> makePhotoCalib(daf::base::PropertySet &metadata, bool strip = false);
 
+/**
+ * Construct a PhotoCalib from the deprecated `Calib`-style fluxMag0/fluxMag0Err values.
+ *
+ * This provides backwards compatibility with the obsoleted Calib object that PhotoCalib replaced.
+ * It should not be used outside of tests that compare with old persisted Calib objects.
+ *
+ * @param fluxMag0 The instrumental flux at zero magnitude. If 0, the resulting `PhotoCalib` will have
+ *                 infinite calibrationMean and non-finite (inf or NaN) calibrationErr.
+ * @param fluxMag0Err The instrumental flux at zero magnitude error. If 0, the resulting `PhotoCalib` will
+ *                    have 0 calibrationErr.
+ *
+ * @returns Pointer to the constructed PhotoCalib.
+ */
+std::shared_ptr<PhotoCalib> makePhotoCalib(double fluxMag0, double fluxMag0Err);
```

Comment:
Just noticed the distinction between "flux" and "calibration". :dizzy_face: Now I think even "ZeroPoint" might not be explicit enough...

Reply:
What about `makePhotoCalibFromDeprecatedCalibZeroPoint()`? It's long, and pretty explicit. I don't want this interface to be too user-friendly, anyway, since it should only be used for backwards compatibility.